### PR TITLE
feat: add toast if bounty creation fails

### PR DESCRIPTION
**Pull request by okhot**
## Issue ticket number and link #1128
https://www.loom.com/share/9b03f745900e4326a3b65b0331a76577?sid=c9c0ac1c-7734-4f44-8020-c56c0977954c

- [X] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested on Chrome and Firefox
- [X] I have tested on a mobile device
- [X] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by okhot** [comment]
@ecurrencyhodler @elraphty @kevkevinpal Made the changes and implemented the proposed implemetation. Can I get a review on this?

File: src/people/main/FocusView.tsx [hunk 1]
```diff
@@ -282,19 +283,33 @@ function FocusedView(props: FocusViewProps) {
...
+          id: '1',
+          title: 'Something went wrong! Unable to create bounty'
+        }
+      ]);
 
-    if (notEdit === true) {
-      setLoading(false);
+      setTimeout(() => {
+        handleFormClose();
+      }, 6000);
```
Comment (kevkevinpal) [inline src/people/main/FocusView.tsx:310]: not sure why we are using `setTimeout` I don't think we should close out of the bounty creation window if creating the bounty fails

File: src/people/widgetViews/postBounty/PostModal.tsx [hunk 2]
```diff
@@ -31,15 +32,21 @@ export const PostModal: FC<PostModalProps> = observer(
 
     const getBountyData = useCallback(async () => {
       const response = await main.getPeopleBounties();
```
Comment (kevkevinpal) [inline src/people/widgetViews/postBounty/PostModal.tsx:34]: why isnt this in the try catch?

File: src/people/main/FocusView.tsx [hunk 3]
```diff
@@ -282,19 +283,33 @@ function FocusedView(props: FocusViewProps) {
...
+
+      if (notEdit === true) {
+        setLoading(false);
+      }
+      if (
+        ui?.meInfo?.hasOwnProperty('url') &&
+        !isNotHttps(ui?.meInfo?.url) &&
+        props?.ReCallBounties
+      )
+        props?.ReCallBounties();
```
[the remark this answers is not in the document]
Comment (okhot) [reply src/people/main/FocusView.tsx:299]: I didn't add these line 🙏🏽

File: src/people/widgetViews/postBounty/PostModal.tsx [hunk 4]
```diff
@@ -23,6 +23,7 @@ export const PostModal: FC<PostModalProps> = observer(
     const isMobile = useIsMobile();
     const [focusIndex, setFocusIndex] = useState(-1);
     const history = useHistory();
+    const [recallBouties] = useState(false);
```
Comment (kevkevinpal) [inline src/people/widgetViews/postBounty/PostModal.tsx:26]: I'm confused why we are adding this state value if it is always going to be `false`

**Comment by okhot** [comment]
@kevkevinpal can you take another look?

File: src/people/widgetViews/postBounty/PostModal.tsx (hunk 4 above)
Comment (okhot) [reply src/people/widgetViews/postBounty/PostModal.tsx]: changed the approach

**Comment by okhot** [comment]
@kevkevinpal @elraphty @ecurrencyhodler Can I please get a review here

**Comment by elraphty** [comment]
@okhot please add a test to this PR

**Comment by okhot** [comment]
> @okhot please add a test to this PR

@elraphty Done ✅

**Pull request merged**